packages/init/src/index.ts

```diff
@@ -48,7 +48,7 @@ async function init() {
     Tracker.get().identifyUser(user.profile.shortId, user.profile.intercomHash);
   }
 
-  Tracker.get().trackInitIdentify(maybeRepositorySubdomain);
+  void Tracker.get().trackInitIdentify(maybeRepositorySubdomain);
```

Comment:
We shouldn't update the repository here, because it hasn't been validated yet.

Ex: If the user does not belong to the repository or is not an Admin, he will have to select another repository in the Step `chooseOrCreateARepository`.

Reply:
I think Identifying the user is One things, updating the repo in the internal tracker is something else.